devices/Mpu6886/Mpu6886.cs

```diff
@@ -0,0 +1,176 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Buffers.Binary;
+using System.Device.I2c;
+using System.Device.Model;
+using System.IO;
+using System.Numerics;
+using System.Threading;
+
+namespace Iot.Device.Mpu6886
+{
+    /// <summary>
+    /// Mpu6886 accelerometer and gyroscope
+    /// </summary>
+    [Interface("Mpu6886 accelerometer and gyroscope")]
+    public class Mpu6886AccelerometerGyroscope : IDisposable
+    {
+        private I2cDevice _i2c;
+        private const double gyroscopeResolution = 2000.0 / 32768.0;
```

Comment:
For the naming conventions, please use Capital letter for constants. So in this case 'GyroscopeResolution'. Same for the next one.
A good practice as well is to place a little remark on where those constants may come from.

Reply:
Fixed capitalization & added description for constants.